Dockerfile

```diff
@@ -32,11 +32,13 @@ RUN go mod download
 # Copy the sources
 COPY ./ ./
 
+RUN go build .
```

Comment:
Mind adding a comment as to the motivation for splitting this out?

Reply:
yes :) good idea